lantern/git/homework.py

```diff
@@ -127,7 +139,8 @@ def alphabet() -> dict:
         alphabet()
         >>> {"a": 1, "b": 2 ...}
     """
-    pass
+    import string
+    return dict(zip(range(1, 27), string.ascii_lowercase))
```

Comment:
Suppose in the English alphabet was added one new letter. Then you will need to change your code. Do this function better. Please do not hardcode some values, like in `range`